sw/device/silicon_creator/meson.build

```diff
@@ -10,6 +10,116 @@ mask_rom_tests = {
   # },
 }
 
+foreach sw_test_name, sw_test_info : mask_rom_tests
```

Comment:
I think this is covered in `sw/device/silicon_creator/testing/meson.build`. The target for verilator is for instance `FLASH_BIN_TEST=build-bin/sw/device/silicon_creator/testing/sw_silicon_creator_lib_driver_keymgr_functest_sim_verilator.elf`

Reply:
yep i've removed this since it is already done.
I'm not sure why i couldn't find this the first time around...